src/events/components/SearchModule/index.tsx

```diff
@@ -28,41 +29,63 @@ const SearchModule: FC = () => {
       )
     );
 
+  const handleToDateClick = (day: Date) => {
+    const clonedDate: Date = new Date(day.getTime());
+    const datetime = DateTime.fromMillis(clonedDate.setHours(0, 0, 0, 0));
```

Comment:
This all seems a bit cumbersome, is there no other way to obtain this result with just `DateTime`?

also, camelCase

Reply:
I changed the input of the function to DateTime now, but I still have to use `const dateTime = day.set({ hour: 0 });`, since the JS-Dates are automatically set to 12 O'clock of the chosen day, while it should be 00:00:00, as that is how our context has them